vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -512,12 +503,12 @@ public Integer playCount() {
     // -----------------------------------------------------------------------------------------------------
     // <editor-fold desc="Files">
     @Nullable
-    public VideoFile getVideoFileForMd5(String md5) {
+    public ProgressiveVideoFile getVideoFileForMd5(String md5) {
```

Comment:
This method is utility method. I think it should be moved to a video utils class.